sdk/elasticsan/Azure.ResourceManager.ElasticSan/src/autorest.md

```diff
@@ -91,12 +92,12 @@ rename-mapping:
   SourceCreationData: ElasticSanVolumeDataSourceInfo
   SnapshotCreationData: SnapshotCreationInfo
   DiskSnapshotList: DiskSnapshotListContent
-  PolicyState: DeleteRetentionPolicyState
-  PreValidationResponse: PreValidationResult
-  VolumeNameList: VolumeNameListContent
-  XMsDeleteSnapshots: DeleteSnapshotsUnderVolume
-  XMsAccessSoftDeletedResources: AccessSoftDeletedVolume
-  XMsForceDelete: ForceDeleteVolume
+  PolicyState: ElasticSanDeleteRetentionPolicyState
+  PreValidationResponse: ElasticSanPreValidationResult
+  VolumeNameList: ElasticSanVolumeNameListContent
+  XMsDeleteSnapshots: ElasticSanDeleteSnapshotsUnderVolume
```

Comment:
This option is now added for this new feature, it exists before. Would this change introduce breaking?

Reply:
yes it would be breaking comparing with the previous version.
But the breaking change detection only compares current code with the last released stable therefore this is fine.

Reply:
`XMsDeleteSnapshots` is a bad name should be fixed